### PR TITLE
Allow running the schedule jobs to run in the foreground.

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
Allows the scheduler to run jobs in the foreground.

### Previous Behavior
Schedule jobs always ran in the background either on a new process or a thread.

### Tests written?

No

### Commits signed with GPG?

Yes

### CODE REVIEW

File: salt/utils/schedule.py [hunk 1]
```diff
@@ -408,23 +408,27 @@ def run_job(self, name):
 
         # Grab run, assume True
         run = data.get('run', True)
+        run_jobs_in_background = self.opts.get('run_jobs_in_background', True)
```
Comment (s0undt3ch) [inline salt/utils/schedule.py]: should it be `run_schedule_jobs_in_background`? On the salt side, `opts` is the **whole** set of options....

Comment (garethgreenaway) [reply salt/utils/schedule.py]: Should probably be `run_schedule_jobs_in_background`.

Comment (s0undt3ch) [reply salt/utils/schedule.py]: Updated

**Review by garethgreenaway — APPROVED** [review]

**Comment by rallytime** [comment]
re-run py3

**Pull request merged**